File: app/src/main/java/com/jawnnypoo/openmeh/service/MehResponse.java
package com.jawnnypoo.openmeh.service;

import com.jawnnypoo.openmeh.data.Deal;
import com.jawnnypoo.openmeh.data.Poll;
import com.jawnnypoo.openmeh.data.Video;

/**
 * Created by John on 4/17/2015.
 */
public class MehResponse {
    Deal deal;
    Poll poll;
    Video video;

    public Deal getDeal() {
        return deal;
    }

    public Poll getPoll() {
        return poll;
    }

    public Video getVideo() {
        return video;
    }

}

File: api/src/main/kotlin/com/jawnnypoo/openmeh/shared/response/MehResponse.kt
package com.jawnnypoo.openmeh.shared.response

import com.jawnnypoo.openmeh.shared.model.Deal
import com.jawnnypoo.openmeh.shared.model.Video
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * The actual response we get when pulling in the meh deal
 */
@Serializable
data class MehResponse(
    @SerialName("deal") var deal: Deal,
    @SerialName("video") var video: Video? = null
)
